stetho/src/main/java/com/facebook/stetho/inspector/database/ContentProviderDatabaseDriver.java

```diff
@@ -13,38 +13,44 @@
 import android.content.Context;
 import android.database.Cursor;
 import android.database.sqlite.SQLiteException;
-import com.facebook.stetho.inspector.protocol.module.Database;
 
-import javax.annotation.concurrent.ThreadSafe;
+import com.facebook.stetho.inspector.protocol.module.Database;
+import com.facebook.stetho.inspector.protocol.module.DatabaseDescriptor;
+import com.facebook.stetho.inspector.protocol.module.DatabaseDriver;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import javax.annotation.concurrent.ThreadSafe;
+
 @ThreadSafe
-public class ContentProviderDatabaseDriver extends Database.DatabaseDriver {
+public class ContentProviderDatabaseDriver
+    extends DatabaseDriver<ContentProviderDatabaseDriver.ContentProviderDatabaseDescriptor> {
 
   private final static String sDatabaseName = "content-providers";
 
   private final ContentProviderSchema[] mContentProviderSchemas;
-  private List<String> mDatabaseNames;
+  private List<ContentProviderDatabaseDescriptor> mDatabaseNames;
   private List<String> mTableNames;
 
-  public ContentProviderDatabaseDriver(Context context, ContentProviderSchema... contentProviderSchemas) {
+  public ContentProviderDatabaseDriver(
+      Context context,
+      ContentProviderSchema... contentProviderSchemas) {
     super(context);
     mContentProviderSchemas = contentProviderSchemas;
   }
 
   @Override
-  public List<String> getDatabaseNames() {
```

Comment:
Oops, this is not backwards compatible (either source or binary).  I'll fix that.